src/02-crm/contacts.apib

```diff
@@ -73,6 +73,7 @@ Get a list of contacts.
                 + added_at: `2016-02-04T16:44:33+00:00` (string)
                 + updated_at: `2016-02-05T16:44:33+00:00` (string)
                 + `web_url`: `https://app.teamleader.eu/contact_detail.php?id=2a39e420-3ba3-4384-8024-fa702ef99c9f` (string)
+                + `national_identification_number`: BE 4283 (string)
```

Comment:
Not sure if the example is a good one, since I have no idea what a national identification number looks like, but this might be slightly confusing with the taxcode here.

Reply:
We could pick one in a format mentioned on https://en.wikipedia.org/wiki/National_identification_number#Europe

I.e. in Spain it's in the format `00000000-A`, where _0 is a digit and A is a checksum letter_.